auto_nag/scripts/regressor.py

```diff
@@ -0,0 +1,96 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+from auto_nag import logger
+from auto_nag.bugbug_utils import BugbugScript
+from auto_nag.utils import get_login_info, nice_round
+from bugbug.models.regressor import RegressorModel
+from bugbug import db, repository
+import hglib
```

Comment:
nit: add hglib to requirements (even if we already have it thanks to bugbug deps)

Reply:
Done in 76a8810fa3ae441ccae2139aeb71b7b13a36feb2.